### PR TITLE
Config devise

### PR DESCRIPTION
**Pull request by victor-soto**

### CODE REVIEW

File: config/environments/test.rb [hunk 1]
```diff
@@ -38,5 +38,5 @@
   config.active_support.deprecation = :stderr
 
   # Raises error for missing translations
-  # config.action_view.raise_on_missing_translations = true
+  # config.action_view.raise_on_missing_translations = true
```
Comment (moisesnandres) [inline config/environments/test.rb]: Remove blank space

**Comment by moisesnandres** [comment]
@cxasper try this

File: app/views/devise/registrations/new.html.haml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+%h2 Sign up
+= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
+  = devise_error_messages!
+  .field
+    = f.label :first_name
+    %br/
+    = f.email_field :first_name, autofocus: true, autocomplete: "first_name"
```
Comment (moisesnandres) [inline app/views/devise/registrations/new.html.haml]: should be a text field

File: app/views/devise/registrations/new.html.haml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  .field
+    = f.label :last_name
+    %br/
+    = f.email_field :last_name, autofocus: true, autocomplete: "last_name"
```
Comment (moisesnandres) [inline app/views/devise/registrations/new.html.haml]: should be a text field

File: app/views/devise/registrations/new.html.haml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  .field
+    = f.label :role
+    %br/
+    = f.email_field :role, autofocus: true, autocomplete: "role"
```
Comment (moisesnandres) [inline app/views/devise/registrations/new.html.haml]: should be a text field

File: app/views/devise/registrations/new.html.haml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  .field
+    = f.label :user_identifier
+    %br/
+    = f.email_field :user_identifier, autofocus: true, autocomplete: "user_identifier"
```
Comment (moisesnandres) [inline app/views/devise/registrations/new.html.haml]: should be a text field

Comment (victor-soto) [reply app/views/devise/registrations/new.html.haml]: done!

File: app/views/devise/registrations/new.html.haml (hunk 4 above)
Comment (victor-soto) [reply app/views/devise/registrations/new.html.haml]: done!

File: app/views/devise/registrations/new.html.haml (hunk 3 above)
Comment (victor-soto) [reply app/views/devise/registrations/new.html.haml]: done!

File: app/views/devise/registrations/new.html.haml (hunk 2 above)
Comment (victor-soto) [reply app/views/devise/registrations/new.html.haml]: done!

**Pull request merged**